crates/configuration/src/network.rs

```diff
@@ -324,6 +325,48 @@ impl NetworkConfigBuilder<Initial> {
         Self::default()
     }
 
+    /// uses the default options for both the relay chain and the nodes
+    /// the only required fields are the name of the nodes,
+    /// and the name of the relay chain ("rococo-local", "polkadot", etc.)
+    pub fn with_chain_and_nodes(
+        relay_name: &str,
+        node_names: Vec<String>,
+    ) -> NetworkConfigBuilder<WithRelaychain> {
+        let mut errors: Vec<anyhow::Error> = vec![];
+
+        if node_names.is_empty() {
```

Comment:
We can remove all the `is_empty` checks now. :)

Reply:
yepp, will do! sounds great

Reply:
Awesome, thanks!!